src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -176,9 +176,7 @@ if(CLR_CMAKE_TARGET_UNIX)
 endif(CLR_CMAKE_TARGET_UNIX)
 
 function(install_library_and_symbols targetName)
-    install_symbols(${targetName} .)
-    set(install_source_file $<TARGET_FILE:${targetName}>)
-    install(PROGRAMS ${install_source_file} DESTINATION .)
+    install_with_stripped_symbols(${targetName} .)
 endfunction()
```

Comment:
A nit: since there is not much left in this function, would it make sense to inline this at call sites by introducing a variable in this file:

```cmake
set(CLR_LIBRARIES_UNIX_ROOT ${CMAKE_CURRENT_LIST_DIR})
```

and at call sites: `install_with_stripped_symbols(System.Native.XYZ ${CLR_LIBRARIES_UNIX_ROOT})`?

Reply:
The . doesn't represent the CMAKE_CURRENT_LIST_DIR, but rather the installer output root (CMAKE_INSTALL_PREFIX), as the destination argument of the install is relative to that. 
But it seems to make sense to replace this function with call to `install_with_stripped_symbols(System.Native.XYZ .)` to make it more consistent with the other installation commands that strip symbols (except for the coreclr one that's special).

Reply:
> replace this function with call to `install_with_stripped_symbols(System.Native.XYZ .)`

Makes sense, I think that will be a bit more work to get packaging pick it up. I tried that previous but the packaging step was failing, IIRC.

Reply:
I am not sure why should there be a difference. It should generate exactly the same files at the same places as before my change.

Reply:
I can see the problem - the install_with_stripped_symbols uses install(PROGRAMS ...) while the installer was using install(TARGETS ...) and I've missed that difference. That causes the .lib for .dll files to not to be installed.